### ISSUE TITLE
Certain Themes Obscure Number of Messages in Conversation

### ISSUE THREAD

**Issue by ryanleesipes** [elementary/mail#12]
The below circles should be a "2" and a "4" respectively, but they are obscured. I'm not sure what is causing this, but it makes for a not very great experience. This problem exists across multiple themes according to coworkers. Not sure if this problem exists in any other software, I have not yet observed it.

![screenshot from 2017-04-26 18-12-34](https://cloud.githubusercontent.com/assets/2141621/25462430/1ec7f58c-2aac-11e7-8cfa-671bcbb2273c.png)

**Comment by HolgerSontag** [comment]
I can confirm this. It goes for all default Gnome 3 Themes as well as all Numix Themes. it gets even worse though. as yxou can see in the screenshot the numbers next to the mailboxes don't work properly.
![selection_008](https://user-images.githubusercontent.com/29298555/26962541-26d32816-4ce7-11e7-8d0d-ce24040b095d.png)
The really strange part is, that if I move the mouse over them upward, all numbers appear perfectly fine (see last entry on the screenshot), if I move the mouse in any other way they morph into a blank box or completely disappear. The Numbers for conversations will also display correctly if I click on them but disappear again once I let go. 
So basically, the application can display things correctly under very specific circumstances. This gets really annoying as it removes the necessary clarity.

**Comment by cassidyjames** [comment]
Does @elementary/ux know if this would be a theme or Granite bug, or is it actually a bug in Mail?

**Comment by HolgerSontag** [comment]
Well, seeing that really only Mail has these issues it would strongly believe that this is a bug in Mail and at least not directly an issue with the theme itself. If you are planning on spreading out the application further than elementaryOS it would make sense to at look into it either way.

**Comment by danrabbit** [comment]
Mail currently uses a custom gtk cell renderer for this conversation list and the code is dark and full of terrors. Corentin is working on a new widget for this and yeah it is an issue in Mail

**Comment by ryanleesipes** [comment]
@danrabbit following your thread about the new widget, is this new widget expected to be present in the first version of the rewrite?

**Comment by danrabbit** [comment]
Yup already is :+1:

**Comment by danrabbit** [comment]
Thanks for your report! We're doing a from-scratch rewrite of Mail and this issue no longer applies to the new code base. I'm closing this report since the issue can't be reproduced in the new version of Mail.
